oper8/x/utils/deps_annotation.py

```diff
@@ -114,6 +114,7 @@ def get_deps_annotation(
     session: Session,
     dependencies: List[Union[dict, Tuple[str, str]]],
     resource_name: str = "",
+    namespace: Optional[str] = None,
```

Comment:
This is minor but we should probably use `_SESSION_NAMESPACE` to distinguish between the current namespace and cluster level resources

Reply:
Good catch!